schema-extended/arxiv-extended.json

```diff
@@ -110,16 +110,6 @@
             },
             "minItems": 1,
             "uniqueItems": true
-          },
-          "cnum": {
-            "title": "CNUM identifier of the conference",
-            "description": "The CNUM is based on the starting day of the conference, with an extra number appended to distinguish conferences starting on the first day.",
-            "type": "string",
-            "pattern": "^C\\d\\d-\\d\\d-\\d\\d(\\.\\d+)?$",
-            "examples": [
-              "C87-12-25",
-              "C87-12-25.2"
-            ]
           }
```

Comment:
I removed this INSPIRE-specific conference identifier. We could replace this with something more generic like `conference_pid` in the spirit of [this proposal](https://www.project-freya.eu/en/blogs/blogs/towards-persistent-identification-of-conferences), though it's not clear whether the proposal is moving forward.

Reply:
We should add something generic in this case, not to miss this part later.